src/Listener/AddDiscussionSubscriptionAttribute.php

```diff
@@ -18,7 +18,7 @@ public function handle(Serializing $event)
     {
         if ($event->isSerializer(DiscussionSerializer::class)
             && ($state = $event->model->state)) {
-            $event->attributes['subscription'] = $state->subscription ?: false;
+            $event->attributes['subscription'] = $state->subscription ?: null;
```

Comment:
Is `?: null` still needed ? Unless there are other falsy values in the database it's now redundant.

Reply:
I assume it's to make sure `null` is the output and not something falsy. After all, this is about having the same data type returned.

Reply:
Yes I suppose so. But if it's now unified between the database and frontend, it means we should be able to pass one value to the other side without any type of casting.

I think it would be safe to remove that null check.

Unless there was some previous version of Flarum that might have saved an empty string or "0" in there?